src/actions/initStore.ts

```diff
@@ -4,22 +4,33 @@ import { Connection } from '../Connection';
 import { sendTransaction } from './transactions';
 import { SetStore, Store } from '@metaplex-foundation/mpl-metaplex';
 
-interface IInitStoreParams {
+/** Parameters for {@link initStore} **/
+export interface InitStoreParams {
   connection: Connection;
+  /** Administrator wallet for the store **/
   wallet: Wallet;
+  /**
+   * - `true`: anyone can list on the store
+   * - `false`: only whitelisted creators can list on the store
```

Comment:
please add a link to https://docs.metaplex.com/architecture/deep_dive/metaplex#whitelistedcreator